pkg/authorize/jwt/claims.go

```diff
@@ -3,7 +3,7 @@ package jwt
 import (
 	"time"
 
-	"gopkg.in/square/go-jose.v2/jwt"
+	"github.com/go-jose/go-jose/v3/jwt"
```

Comment:
if we could continue using a v2 version, it'd be great.
Have you tried with `gopkg.in/go-jose/go-jose.v2`?

Reply:
done, I had issues in with v2 and I chose v3 in master but we can fix it here with v2.